lists/dataframe.py

```diff
@@ -0,0 +1,27 @@
+# ------------------------------------------------------------------------------------
+# Tutorial: creating a dataframe from lists
+# ------------------------------------------------------------------------------------
+# Think of a dataframe as an excel spreadsheet. All it is is a two-dimensional structure consisting of rows and columns.
```

Comment:
Maybe you can add what you should install before to talk about the import ? 
In this way you describe all the steps, it will be helpful for your tutorial and we will see on our side for the github action if no one want to contribute on this part

Reply:
okay